src/multithreader.cpp

```diff
@@ -0,0 +1,26 @@
+#include <iostream>
```

Comment:
This is a pretty epic file. Not familiar with c++ so not sure what it does but it looks epic